src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForPathAndValueEntity.cs

```diff
@@ -75,7 +78,32 @@ public ICommand SetValueOrigin(TBuildingBlock buildingBlock, ValueOrigin valueOr
       {
          return new UpdateValueOriginInPathAndValueEntityCommand<TBuilder>(pathAndValueEntity, valueOrigin, buildingBlock).Run(Context);
       }
-      
+
+      public IMoBiCommand ConvertDistributedParameterToConstantParameter(TBuilder distributedParameter, TBuildingBlock buildingBlock, IReadOnlyList<TBuilder> subParameters)
+      {
+         var meanEntity = subParameters.FindByName(Constants.Distribution.MEAN);
```

Comment:
Why do we need to get the mean parameter and then create a constant parameter with `meanEntity.Value` instead of with `distributedParameter.Value`?
Especially because
a) not all distributions have a "Mean" sub-parameter (e.g. uniform distribution)
b) the value of the distributed parameter is not necessarily the mean value of the underlying distribution

Reply:
This is not a real 'Parameter' or 'DistributedParameter'. It's just a collection of 'PathAndValueEntity' like 'ParameterValues'. So when converting this, there is no value in the `Value` property, nor a formula in the `Formula` field. Both are set to null. I believe the expectation is that the parameter is created when the model is constructed and the value will be calculated at that time.

This initial view of distributed parameter is arranged by the convention that the paths are coincidental for Mean/Percentile, or whatever subparameters are there.

So, not sure how to proceed with that. I could add other conventions than `Mean`

Reply:
The type is stored there.

Reply:
Yep, that was exactly what I thought. Create temporary parameter with the proper distribution and percentile and get its value, then save it